### ISSUE TITLE
Requesting App Ratings

### ISSUE THREAD

**Issue by anubhavpulkit** [systers/PowerUp#334]
## Description
As a user If I want to give a rating this app, I need to go to the app store to give ratings.
I need to displays an in-app prompt that asks for a rating.

With the help of apple official API StoreKit we call the SKStoreReviewController An object that controls the process of requesting App Store ratings and reviews from users.

In which the system offers a consistent, nonintrusive way for apps to request ratings. To use this feature, you simply identify places in your app's user experience where it makes sense to ask for feedback. If the user hasn't already given feedback, the system displays an in-app prompt that asks for a rating. The user can supply feedback or dismiss the prompt with a single tap. 

## Mocks
![review1](https://user-images.githubusercontent.com/47811606/73873532-c7494f80-4877-11ea-8015-1063072ffe1f.png)

### Update [Required]
- [displays app prompt that asks for a rating]
- [conforms to Apple's Human Interface Guidelines for requesting ratings and reviews]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
[2] hours

**Comment by sunjunkie** [comment]
@anubhavsingh16 This new issue must be approved before you work on the fix or submit a PR. See
https://github.com/anitab-org/powerup-iOS/wiki/How-to-Contribute

**Comment by anubhavpulkit** [comment]
@sunjunkie I get approval in IRC.

**Comment by sunjunkie** [comment]
@anubhavsingh16 please review the guidelines for  contributing to Powerup-iOS https://github.com/anitab-org/powerup-iOS/wiki/How-to-Contribute

**Comment by anubhavpulkit** [comment]
@sunjunkie  Ok Sorry, so what I have to do now? May I delete my Pull Request?

**Comment by sunjunkie** [comment]
Yes, go ahead. You can also expand the issue description to include info about Apple guidelines and then add those requirements to the Update requirements. After that I’ll review for approval.

**Comment by anubhavpulkit** [comment]
> Yes, go ahead. You can also expand the issue description to include info about Apple guidelines and then add those requirements to the Update requirements. After that I’ll review for approval.

ok

**Comment by anubhavpulkit** [comment]
@sunjunkie Please review my issue.

**Comment by sunjunkie** [comment]
I approve this issue. (I added a requirement to the Update section.)

**Comment by anubhavpulkit** [comment]
> I approve this issue. (I added a requirement to the Update section.)

Ok, Thank You.
